x/dsmr/storage.go

```diff
@@ -40,10 +41,9 @@ func (NoVerifier[T]) Verify(Chunk[T]) error {
 }
 
 type StoredChunkSignature[T Tx] struct {
-	Chunk          Chunk[T]
-	LocalSignature NoVerifyChunkSignatureShare // Decouple signature share / certificate types
```

Comment:
We removed this because if we want no-verify behavior we can achieve this by changing how we handle `Cert` through an interface